source/hic_hal/stm32/stm32h743ii/extended_features/include/adapter.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _ADAPTER_H_INCLUDED_
+#define _ADAPTER_H_INCLUDED_
+
+typedef enum
+{
+    ADAPTER_UDB_13_FLEX,
+    ADAPTER_JOKER_FLEX,
+    ADAPTER_UDB_6_FLEX,
+    ADAPTER_UDB_17_FLEX,
+    ADAPTER_UDB_12_FLEX,
+    ADAPTER_UNKNOWN
+} adapter_type_t;
+
+#define ADAPTER_UDB_13_FLEX_ADC_MIN     (54000)
```

Comment:
Would this be better in a table of some sort to allow easy iteration?

    // Table to map adapter type to min ADC value that can be read for type
    static const uint16_t s_adapter_type_adc_map[] = {
        [ADAPTER_UDB_13_FLEX] = 54000,
        [ADAPTER_UDB_JOKER_FLEX] = 51000,
        ...
    };

Also might be better to put it into the C file where it's used, since I suspect it will only be one file that needs this info.

Reply:
Good suggestion, see this syntax the first time.